### PR TITLE
Update work for issue 263 to connect to backend

### PR DESCRIPTION
**Pull request by humphd**
We merged a bunch of code today to get the redis and pipeline connected, and added it to the front-end, based on @lucacataldo's work from #346.  This needs more work, but IT WORKS! :fire:

Screen Shot 2019-12-03 at 1 34 02 PM

### CODE REVIEW

File: src/backend/utils/storage.js [hunk 1]
```diff
@@ -34,6 +34,8 @@ module.exports = {
       .hmset(
         // using guid as keys as it is unique to posts
         post.guid,
+        'guid',
+        post.guid,
```
Comment (cindyledev) [inline src/backend/utils/storage.js]: #427 just landed and it made these changes. This is no longer needed

Comment (humphd) [reply src/backend/utils/storage.js]: Good catch.  I've rebased and updated this, new commits pushed to this branch.

**Comment by cindyledev** [comment]
Pretty excited to see this land

**Comment by humphd** [comment]
I fixed a few more small things, and now posts have content:

Screen Shot 2019-12-03 at 3 09 26 PM

**Comment by humphd** [comment]
Let's not merge this yet, it's just a WIP to get us moving.

**Pull request merged**